docker/Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM ubuntu:xenial
+FROM alpine:3.6
 
-RUN apt-get update -y && apt-get upgrade -y && apt-get install -y \
-build-essential \
+RUN apk upgrade && apk update && apk add \
+g++ \
+make \
 cmake \
-libboost-dev \
-libboost-filesystem-dev \
-libboost-program-options-dev \
-libboost-regex-dev \
-libsqlite3-dev \
+boost-dev \
+boost-filesystem \
```

Comment:
Even though this package is downloaded, I am getting these errors when executing `docker run`:

> [ 42%] Building CXX object CMakeFiles/test_validator_v41.dir/test/main_test.cpp.o
[ 44%] Building CXX object CMakeFiles/test_validator_v42.dir/test/main_test.cpp.o
make[2]: *** No rule to make target '/usr/lib/x86_64-linux-gnu/libboost_filesystem.a', needed by 'bin/test_validator_v41'.  Stop.
make[2]: *** Waiting for unfinished jobs....
[ 46%] Building CXX object CMakeFiles/test_validator_v41.dir/test/vcf/parser_v41_test.cpp.o
make[2]: *** No rule to make target '/usr/lib/x86_64-linux-gnu/libboost_filesystem.a', needed by 'bin/test_validator_v42'.  Stop.
make[2]: *** Waiting for unfinished jobs....
[ 48%] Building CXX object CMakeFiles/test_validator_v42.dir/test/vcf/parser_v42_test.cpp.o
make[1]: *** [CMakeFiles/Makefile2:107: CMakeFiles/test_validator_v41.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:220: CMakeFiles/test_validator_v42.dir/all] Error 2
make: *** [Makefile:95: all] Error 2

Reply:
Strange! I am not getting this error. Everything was successful on my system, along with the vcf_validator, vcf_debugulator etc.
I will look into it.

Reply:
Solved, it was an issue with the CMake cache... 😓